package.json

```diff
@@ -11,7 +11,7 @@
     "start-dev": "nodemon server/server.js",
     "start": "node server/server.js",
     "seeder": "node server/seeder.js",
-    "test": "jest"
+    "test": "jest --detectOpenHandles"
```

Comment:
I don't think detectOpenHandles is needed, that is just for debugging something funky, are you having issues without it?

Reply:
@allenhsieh  havent taken out, let me check